content/how-to/messaging/microsoftteams/connection/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Microsoft Teams Connection"
+description: ""
+lead: ""
+date: 2023-02-07
+draft: false
+weight: 500
+toc: false
+tags:
+  - microsoft teams
+resources:
+  - src:
```

Comment:
Looks like you forgot to link them here, I honestly am not sure what this does though